docs/core/entity.md

```diff
@@ -91,6 +89,37 @@ The following properties are used and controlled by Home Assistant, and should n
 | ------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | enabled | boolean | `True`  | Indicate if entity is enabled in the entity registry. It also returns `True` if the platform doesn't support the entity registry. Disabled entities will not be added to Home Assistant. |
 
+## Entity class attribute
+
+Writing property methods for each property is just a couple of lines of code,
+for example
+
+```python
+class MySwitch(SwitchEntity):
+
+    @property
+    def icon(self) -> str | None:
+        """Icon of the entity."""
+        return "mdi:door"
+
+    ...
+```
+
+Alternatively, a shorter form is to set Entity class attributes according to the
+following pattern:
```

Comment:
This form is complete new to me, is this the recommended way to do it currently / in the future? In any case, it'd be great to have the preferred way written clearly out in the text.

Reply:
It helps to reduce LOC, but early stages (PR Linked in the description).